README.md

```diff
@@ -60,7 +61,7 @@ conda environment before submitting the the bash script. See
 ### On CRIB
 
 [CRIB](https://crib.utwente.nl/) is the ITC Geospatial Computing Platform. You
-can run the model using `Matlab` or `Octave`. Currently, running the 
+can run the model using `Matlab` or `Octave`. Currently, running the
 exceutable file on CRIB is not supported because MATLAB Runtime can not be
 installed there. See [Install PyStemmusScope](#install-pystemmusscope).
```

Comment:
There are two commands below that uses `pip install` from `main` branch. when pystemmusscope is published on pypi, we can install it by `pip install pystemmusscope`. Shall we fix the commands in this PR?

Reply:
Done!